opencensus-shim/src/main/java/openconsensus/opencensusshim/metrics/Metrics.java

```diff
@@ -48,10 +48,10 @@ public static ExportComponent getExportComponent() {
   /**
    * Returns the global {@link MetricRegistry}.
    *
-   * <p>This {@code MetricRegistry} is already added to the global {@link
+   * <p>This {@code MetricCollection} is already added to the global {@link
```

Comment:
OC-Shim doesn't need to be updated. (previous comment is under wrong file)

Reply:
Done